marthas_dashboard/colors.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
specifying colors constituted a lot of code in the middle of view.py so I moved it to a separate file